ci/docker/conda-python.dockerfile

```diff
@@ -25,7 +25,7 @@ ARG python=3.6
 COPY ci/conda_env_python.yml /arrow/ci/
 RUN conda install -q \
         --file arrow/ci/conda_env_python.yml \
-        python=$PYTHON_VERSION \
+        python=${python} \
```

Comment:
Question: what does the ``ARG python=3.6`` line a little bit above do? that 'python' is not related to the ${python} here?

Reply:
See my comment above and the docker file [reference](https://docs.docker.com/engine/reference/builder/#arg)